jenkins/ci.suse.de/cloud-socok8s.yaml

```diff
@@ -6,4 +6,27 @@
     repo-credentials: c2350527-476a-45df-b406-84f028614682
     jobs:
       - '{name}-integration'
-      - '{name}-nightly-airship-from-rpm'
+
+- project:
```

Comment:
Hm. Now we introduce variables that are only relevant for the file `jenkins/ci.suse.de/pipelines/socok8s-airship-from-rpm.Jenkinsfile`. And not the `Jenkinsfile` from the suse-cloud/socok8s repo.
I think this will be confusing. I would still be in favor to have the jenkinsfiles in the socok8s repo...Anyway not blocking this PR...